src/js/login/containers/Main.jsx

```diff
@@ -14,7 +14,6 @@ class Main extends React.Component {
     this.setMyToken = this.setMyToken.bind(this);
     this.getLogoutUrl = this.getLogoutUrl.bind(this);
     this.getLoginUrl = this.getLoginUrl.bind(this);
-    this.getVerifyUrl = this.getVerifyUrl.bind(this);
```

Comment:
Did you mean to remove this from the original Main component in `login`?

Reply:
Good catch will add this in. It's also included in the login component but we need it for this component too in case a user goes straight to the /signin page (the header nav would never have been loaded in this case)

Reply:
I misspoke; I looked through and did mean to remove this. we only need this one in the `Main` container in the `signin` folder because only that one uses the VerifyScreen where this is applicable. There is no need to fetch the verify URL for the `login` folder because it is never used

Reply:
Shouldn't we probably keep this around anyway, at least for soft launch?